index.js

```diff
@@ -16,6 +16,10 @@ const TEN_MEGABYTES = 1000 * 1000 * 10;
 function handleArgs(cmd, args, opts) {
 	let parsed;
 
+	if (opts && opts.env && opts.envExtend !== false) {
+		opts.env = Object.assign(process.env, opts.env);
+	}
```

Comment:
I did this as the first part of the `handleArgs` in the `Object.assign` block at line `36` (or after that block) because I wasn't sure that `cross-spawn` isn't using that variable internally. But if you want to move it, feel free to suggest so.